docs/README.md

```diff
@@ -20,6 +20,9 @@ The Editor offers rich new value to users with visual, drag-and-drop creation to
 
 ## Quick links
 
+### Gutenberg Principles
+[The Gutenberg principles document](/docs/explanations/principles.md) highlights a few principles that are guiding the overall project and shape various development and design decisions.
+
```

Comment:
Can we move this section into the `Contribute to the block editor` below, I don't think it should be the first Quick Link.

This is the first page that shows when viewing the [Block Editor Handbook](https://developer.wordpress.org/block-editor/) and people are going to be more interested in how to build for and extend first.

With that also said, I think a better more prominent spot would be the repository top-level readme, as that is only showed within the repo and more likely target to developers of the project.

Reply:
I'm going to go ahead and move this into the repo top-level README.md - I agree it makes more sense to target contributors. 

Given your [comment here however](https://github.com/WordPress/gutenberg/pull/30963#pullrequestreview-643571178), do you think I still need to expose this on on the built docs and if so, where do you think it should be in the docs structure?

Reply:
See 219f0af (and 1a648ba where wording was changed a bit).